### PR TITLE
108 make output messages optional

### PR DESCRIPTION
**Pull request by wcornwell**
Working on #108 and #194

### CODE REVIEW

File: R/align_taxa.R [hunk 1]
```diff
@@ -149,29 +157,29 @@ align_taxa <- function(original_name,
     # take unique values so each name only processed once
     dplyr::filter(!duplicated(original_name))
   
-  if (all(taxa$tocheck$checked)) {
+  if (all(taxa$tocheck$checked)|all(is.na(taxa$tocheck$checked))) {
     message("  - all taxa are already checked, yay!")
     return(invisible(taxa$tocheck))
```
Comment (fontikar) [inline R/align_taxa.R:162]: Can we also have a progress bar for alignment please?

**Review by fontikar — COMMENTED** [review]
Minor request for a progress bar for `align_taxa` otherwise it looks great! I like the `output` argument!

**Comment by fontikar** [comment]
@wcornwell if you merge in `develop` into `108-make-output-messages-optional ` macOS should pass

File: R/align_taxa.R (hunk 1 above)
Comment (wcornwell) [reply R/align_taxa.R:162]: yes, but maybe needs to be part of #191 ?

**Review by fontikar — APPROVED** [review]

**Pull request merged**